src/opm/output/eclipse/LoadRestart.cpp

```diff
@@ -1218,10 +1223,8 @@ namespace Opm { namespace RestartIO  {
             restoreExtra(rst_view, extra_keys, es.getUnits(), rst_value);
         }
 
-        return {
-            std::move(rst_value),
-            restore_cumulative(rst_view, schedule)
-        };
+        restore_cumulative(summary_state, rst_view, schedule);
+        return std::move(rst_value);
```

Comment:
I didn't notice this on the first go.  Since the `rst_value` object has the correct type this can probably be simplified to just
```C++
return rst_value;
```
to activate the [NRVO](https://en.cppreference.com/w/cpp/language/copy_elision).